packages/demo/src/index.ts

```diff
@@ -87,3 +91,19 @@ dragContainer.addEventListener('drop', (event) => {
 dragContainer.addEventListener('dragover', (event) => {
   event.preventDefault();
 });
+
+strokeWidthInputInteractive.addEventListener('change', () => {
+  if (strokeWidthInputInteractive) {
+    const context = canvasNode.getContext('2d');
+    context?.clearRect(0, 0, canvasNode.width, canvasNode.height);
+    canvasOutliner(canvasNode, './pikachu.png', +strokeWidthInputInteractive.value, strokeColorInputInteractive.value);
```

Comment:
seems like it'll only work with pikachu.png since it's hardcoded
but we need to make it work with any uploaded pic :(

Reply:
Fixed